src/client/pages/store-management/store-management-view.html

```diff
@@ -0,0 +1,32 @@
+<div class="store-management-view">
+    <div class="container is-fullhd">
+        <section>
+            <div class="hero is-primary">
+                <div class="title is-1">Store Management</div>
```

Comment:
is-1 we use for column gap

Reply:
In bulma, there is is-1 modifier for title. That's why I used there.